From 8024d364bbf11e45573d341458a962c83c3deafd Mon Sep 17 00:00:00 2001
From: Kuaipz <2685697179@qq.com>
Date: Wed, 17 May 2023 14:00:18 +0800
Subject: [PATCH] treble-adb-secure

---
 config/common.mk | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 0e6990a..13faa5d 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -15,17 +15,6 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.com.google.clientidbase=$(PRODUCT_GMS_CLIENTID_BASE)
 endif
 
-ifeq ($(TARGET_BUILD_VARIANT),eng)
-# Disable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
-else
-# Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-
-# Disable extra StrictMode features on all non-engineering builds
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
-endif
-
 # Backup Tool
 PRODUCT_COPY_FILES += \
     vendor/lineage/prebuilt/common/bin/backuptool.sh:install/bin/backuptool.sh \
-- 
2.34.1

