From f0e266f67789015e846c2952b3046b3520c00619 Mon Sep 17 00:00:00 2001
From: Kuaipz <2685697179@qq.com>
Date: Fri, 2 Jun 2023 11:07:43 +0800
Subject: [PATCH] treble-gms-lite

---
 common/common-vendor.mk | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/common/common-vendor.mk b/common/common-vendor.mk
index b5b049a..8f451e4 100644
--- a/common/common-vendor.mk
+++ b/common/common-vendor.mk
@@ -57,9 +57,7 @@ PRODUCT_PACKAGES += \
     GoogleServicesFramework \
     LatinIMEGooglePrebuilt \
     Phonesky \
-    PixelSetupWizard \
     PrebuiltGmsCoreSc \
-    SetupWizardPrebuilt \
     TrichromeLibrary \
     TrichromeLibrary-Stub \
     WebViewGoogle \
@@ -92,10 +90,7 @@ endif
 # FULL GAPPS (expand core gapps)
 ifeq ($(strip $(GAPPS_BUILD_TYPE)),2)
 PRODUCT_PACKAGES += \
-    AndroidAutoStubPrebuilt \
     CalendarGooglePrebuilt \
-    Chrome \
-    Chrome-Stub \
     GoogleTTS \
     GooglePrintRecommendationService \
     LocationHistoryPrebuilt \
@@ -105,7 +100,6 @@ PRODUCT_PACKAGES += \
     PrebuiltDeskClockGoogle \
     SoundPickerPrebuilt \
     TagGoogle \
-    Velvet \
     WellbeingPrebuilt
 endif
 endif
-- 
2.41.0

